src/routes/auth/signin.tsx

```diff
@@ -0,0 +1,10 @@
+import { createFileRoute, redirect } from '@tanstack/react-router';
+
+// legacy redirection for routes using the old `signin` route.
+// this should be removed 6 months after the push of the new client interface in production.
+// estimated date at time of writing is July 2025
```

Comment:
I understand what you mean, but it's a little bit confusing when you read it, I think we can keep it simple as follow:
```suggestion
// this should be removed 6 months after the push of the new client interface in production, approximately around July 2025
```